ruby/alias_manager.rb

```diff
@@ -0,0 +1,62 @@
+# ask user for their full name
+# split the string
+# swap between first and last name
+# create a loop that cheks if letter is vowel or consonant
+# 	-if vowel - create a method that finds the next vowel and call it
+# 	-else use .next to replace the consonant.
+continue = true
+new_names = []
+original_names = []
+counter = 0
+vowels = ['a', 'e', 'i', 'o', 'u']
+consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'y', 'z']
+
+def get_next(arr, letter)
```

Comment:
Here instead of passing one letter at a time in try passing the entire name (or string) in as a single param.  Let your methods do the heavy lifting and let the driver code simply test the methods.